cloud-js-containers/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Easy container example
+# Containers on AWS Fargate
```

Comment:
Isn't this supposed to be cloud-agnostic?

Reply:
Good catch! Although it explicitly states it's a companion to the AWS tutorial. What was the motivation behind `cloud` examples? The others do explicitly use the `pulumi-cloud` package. @justinvp should we add that these examples are using a preview library?